src/ceph_fuse.cc

```diff
@@ -57,6 +62,9 @@ int main(int argc, const char **argv, const char *envp[]) {
     } else if (ceph_argparse_flag(args, i, "--localize-reads", (char*)NULL)) {
       cerr << "setting CEPH_OSD_FLAG_LOCALIZE_READS" << std::endl;
       filer_flags |= CEPH_OSD_FLAG_LOCALIZE_READS;
+    } else if (ceph_argparse_flag(args, i, "-h", "--help", (char*)NULL)) {
+      usage();
+      return 0;
```

Comment:
writing "return 0" here is confusing because it makes it seem like calling this will result in the process exiting with status zero.  What actually happens is that generic_usage calls exit(1), so you should actually assert(0) here because we don't expect to ever return.